package.json

```diff
@@ -11,11 +11,12 @@
   },
   "dependencies": {
     "@heroicons/react": "^1.0.5",
-    "@solana/wallet-adapter-base": "^0.9.3",
-    "@solana/wallet-adapter-react": "^0.15.3",
-    "@solana/wallet-adapter-react-ui": "^0.9.5",
-    "@solana/wallet-adapter-wallets": "^0.15.3",
-    "@solana/web3.js": "^1.31.0",
+    "@noble/ed25519": "^1.7.1",
+    "@solana/wallet-adapter-base": "^0.9.18",
+    "@solana/wallet-adapter-react": "^0.15.21-rc.4",
+    "@solana/wallet-adapter-react-ui": "^0.9.19-rc.4",
+    "@solana/wallet-adapter-wallets": "^0.19.4",
+    "@solana/web3.js": "^1.58.0",
```

Comment:
This version of web3.js has done away with `tweetnacl`, hence the reason for the change to the signature verification elsewhere in this PR.